documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,31 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
+
+. VMware source providers only (All optional):
+
+* *Preserving static IPs of Windows VMs*: By default, Virtual Network Interface Cards (VNICs) change during the migration process. This results in Windows VMs that are set with a static IP in vSphere losing their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, and toggle the *Whether to preserve the static IPs* switch in the window that opens. Then click *Save*.
```

Comment:
I still think the `Windows VNICs` term needs to be replaced but I'm not sure how - either by `Windows VMs` which implies VMs that are installed with Windows operating system, or `VNICs of Windows VMs`.
or we can handle it differently, e.g.:
```suggestion
* *Preserving static IPs of Windows VMs*: By default, Virtual Network Interface Cards (VNICs) change during the migration process. This results in VNICs that are set with a static IP in vSphere losing their IP. To avoid this and preserve static IPs of Windows VMs, click the Edit icon next to *Preserve static IPs*, and toggle the *Whether to preserve the static IPs* switch in the window that opens. Then click *Save*.
```
@RichardHoch what do you think?

Reply:
@ahadas Pretty good -- small tweak --stylistically, either "To avoid this and to preserve.." or "To avoid this by preserving..." would be better. Is one more accurate than the other?